<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GgsCsvGraphDialog</class>
 <widget class="QDialog" name="GgsCsvGraphDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSV图数据加载</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBoxNode">
       <property name="title">
        <string>节点</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="labelNodeFileName">
              <property name="text">
               <string>节点文件：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditNodeFileName"/>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonNodeFileBrowser">
              <property name="text">
               <string>…</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="labelNodeLayerName">
              <property name="text">
               <string>图层名称：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditNodeLayerName"/>
            </item>
            <item>
             <widget class="QLabel" name="labelNodeFileSpilter">
              <property name="text">
               <string>分隔符：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxNodeFileSpliter">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="labelNodeX">
              <property name="text">
               <string>X坐标：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxXHeaders">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelNodeY">
              <property name="text">
               <string>Y坐标：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxYHeaders">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="labelNodeId">
              <property name="text">
               <string>标识字段：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxIdHeaders">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="labelSelectNodeHeaders">
              <property name="text">
               <string>请勾选需要保留的节点属性：</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxAllNodeHeaders">
              <property name="text">
               <string>全选</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="listWidgetNodeHeaders"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="labelCRS">
              <property name="text">
               <string>请设置图层CRS：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditCRS"/>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonCRS">
              <property name="text">
               <string>…</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxRelation">
       <property name="title">
        <string>连边</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="labelEdgeFileName">
              <property name="text">
               <string>连边文件：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditEdgeFileName"/>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonEdgeFileBrowser">
              <property name="text">
               <string>…</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="labelEdgeLayerName">
              <property name="text">
               <string>图层层名：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditEdgeLayerName"/>
            </item>
            <item>
             <widget class="QLabel" name="labelEdgeFileSpilter">
              <property name="text">
               <string>分隔符：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxEdgeFileSpliter">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="labelEdgeSource">
              <property name="text">
               <string>源：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxEdgeSourceHeaders">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelEdgeTarget">
              <property name="text">
               <string>宿：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxEdgeTargetHeaders">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>47</width>
                <height>18</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="labelSelectEdgeHeaders">
              <property name="text">
               <string>请勾选需要保留的连边属性：</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxAllEdgeHeaders">
              <property name="text">
               <string>全选</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="listWidgetEdgeHeaders"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>131</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okButton">
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>GgsCsvGraphDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
